packages/calypso-products/test/get-all-products-for-plan.ts

```diff
@@ -0,0 +1,33 @@
+import { getAllProductsForPlan } from '../src';
+
+describe( 'getAllProductsForPlan', () => {
+	test( 'returns an empty array when given an invalid plan', () => {
+		const result = getAllProductsForPlan( 'invalid-plan' );
+		expect( result ).toEqual( [] );
+	} );
+
+	test( 'returns an empty array for Jetpack free', () => {
+		const plan = 'jetpack_free';
+		const result = getAllProductsForPlan( plan );
+		expect( result ).toEqual( [] );
+	} );
+	test( 'Checks for products that should exist in "jetpack_complete" plan', () => {
+		const plan = 'jetpack_complete';
+		const result = getAllProductsForPlan( plan );
+		expect( result ).toContain( `jetpack_backup_t2_yearly` );
```

Comment:
I've got a slight concern that the test might break if we ever update the content of the Complete plan. Maybe write an assertion for one or two products only?

Reply:
fixed